### PR TITLE
Price dialog changes and a couple of bug fixes

### PR DESCRIPTION
**Pull request by Bob-IT**
Create a PR for this lot, it all looks OK to me and does what I intended but thought I would run it pass you before pushing.
Changed the security and price dialogues to use GkWindows which allows them to both be open at the same time just like in 2.6
Fixed Bug797165 and added a gnc-warning for overwriting/replacing existing prices.
The last commit made reading of the log file easier by eliminating false indents based on logger names for when the log.conf file is used, My log.conf is full of these just commented out so I can remember them and use them by removing the comment maker.

### CODE REVIEW

**Comment by Bob-IT** [comment]
I think I may of missed testing a couple of scenarios for Bug797165, will test them later.

**Comment by Bob-IT** [comment]
OK have done more tests for Bug 797165 and can not get it to fail, with the price editor open, tried importing prices which appeared OK and selectable, remove old prices worked, added prices manually and removed them which all updated as expected.

File: gnucash/gnome/dialog-price-editor.c [hunk 1]
```diff
@@ -192,6 +194,56 @@ price_to_gui (PriceEditDialog *pedit_dialog)
...
 
 
+static gboolean
+pedit_dialog_replace_found_price (PriceEditDialog *pedit_dialog,
+                                  const gnc_commodity *commodity,
+                                  const gnc_commodity *currency, time64 t)
+{
+    gboolean price_found = FALSE;
+    GNCPrice *test_price =  gnc_pricedb_lookup_day_t64(pedit_dialog->price_db,
+                                commodity, currency, t);
```
Comment (jralls) [inline gnucash/gnome/dialog-price-editor.c]: Whitespace: Add a space between the function and the (, align commodity with pedit_dialog.

File: gnucash/gnome/dialog-price-editor.c [hunk 2]
```diff
@@ -192,6 +194,56 @@ price_to_gui (PriceEditDialog *pedit_dialog)
...
+                price_found = TRUE;
+        }
+        gnc_price_unref (test_price);
+    }
+
+    if (price_found)
+    {
+        gint response;
+        GtkWidget *dialog;
+        gchar *message = _("You are about to replace an existing price?");
```
Comment (jralls) [inline gnucash/gnome/dialog-price-editor.c]: Seems a bit stilted: "Replace price? You are about to replace an existing price?".
I'd think "Are you sure you want to replace the existing price?" would be more idiomatic.

File: gnucash/gnome/dialog-price-editor.c (hunk 1 above)
Comment (Bob-IT) [reply gnucash/gnome/dialog-price-editor.c]: I must of missed this, normally I would add the space on parts I touch and leave the rest in the file alone, maybe it was like that when I copied and pasted, changed.

File: gnucash/gnome/dialog-price-editor.c (hunk 2 above)
Comment (Bob-IT) [reply gnucash/gnome/dialog-price-editor.c]: Will change text.

**Comment by Bob-IT** [comment]
These changes have been pushed but I must of done the same as last time as this was not closed automatically, next time do a branch will confirm procedure.

**Comment by gjanssens** [comment]
Did you force push the branch here before merging it to maint ?
If you didn't the branch head for this PR and for what got merged in maint are not the same and hence github won't know the PR got merged.

**Comment by Bob-IT** [comment]
I am sure I force pushed but and I know that my local price-changes branch and origin/price-changes were the same but am wondering if after checking that I rebased on upstream/maint and did not force push after that to origin. Like I said the next time I create a branch, I will write down what I do in order in case it happens again, thank you.

**Comment by gjanssens** [comment]
Op vrijdag 26 april 2019 18:46:35 CEST schreef Bob-IT:
> I am sure I force pushed but and I know that my local price-changes branch
> and origin/price-changes were the same but am wondering if after checking
> that I rebased on upstream/maint and did not force push after that to
> origin. Like I said the next time I create a branch, I will write down what
> I do in order in case it happens again, thank you.

A rebase would indeed invalidate the PR as well.

**Pull request closed without merging**